homework.md

```diff
@@ -39,75 +43,91 @@ Using CSS, how would I select every `<h1>` tag on the page?
 
 }
 ```
+h1 {
+
+}
 
 ## Question 7
 
 What is JavaScript?
 
-A programming language used on the internet
+It's a language used to make a website become interactive.
 
 ## Question 8
 
 What three problem does Git & Github solve?
+What three problems does Git & Github solve?
 
-When people want to show off code to each other they can put it on Github
+1. Tracks changes 2.Instant feedback on collaborative work. 3.Version control
 
 ## Question 9
 
 What happens when you `fork` a repository?
 
 You delete it
+Fork is another term for 'copy'.
 
 ## Question 10 
 
 What happens when you clone a repostory?
+What happens when you clone a repository?
 
 It send it to a friend
+Clone is another term for 'paste'.
+
 
 ## Question 11
 
 What is a Pull Request?
 
 When you send a file over the internet
+Pull request is another term for asking others for feedback.
```

Comment:
Pull requests let you tell others about changes you've pushed to a branch in a repository on GitHub. Once a pull request is opened, you can discuss and review the potential changes with collaborators and add follow-up commits before your changes are merged into the base branch.

Reply:
Noted, thanks Jon 👍